chainguard-dev/edu
# 24g minor Academy edits

## Type of change
Each edit is a small edit, though they do add up ! This PR contains a few different commits, primarily adding links, updating formatting, or other small changes to files. I also walked through the cosign materials that we currently have on the Academy, and I made some minor changes to update these. 

### What should this PR do?
Resolves #1706 
Resolves #1711 

### Why are we making this change?
These changes are minor but should make everything look a little nicer :) for the cosign materials, it should alter some commands to be more accurate with their syntax. It also included a pass-over of the article text so some clarifying edits were made there too.

### What are the acceptance criteria? 
I would suggest that the "files changed" section of the PR be focused on, because most of these changes are small. All of the cosign *commands* are unchanged, but styling is a little different (for example, some old commands referenced the sigstore course even though they were on the Academy, so I changed that so they were more relevant). Though, it is welcome that someone would review the commands as well if they wish.

### How should this PR be tested?
Check the deploy preview for the changes, and run any cosign commands locally if need-be. The updated phrasing of the articles should make it easier to follow along.
